### DISCUSSION TITLE
Delete a Space

### DISCUSSION THREAD

**Discussion by minus78** [snapshot-labs/snapshot#2082] [Q&A]
Hello!

I would like to know if it is possible to delete a "Space", once you have created one. I would like to know if it is possible to start a DAO from scratch? Or to have several DAOs with the same ENS?

Thanks

**Comment by metrox-eth** [comment]
To delete a space, we don't have a button in the UI at the time of writing, but you can reach out on the Discord and we will help. https://discord.snapshot.org/
Yes if you use subdomains.

**Comment by jambestwick** [comment]
need a UI website
